exonum/src/encoding/float.rs

```diff
@@ -243,6 +243,13 @@ impl ExonumJson for F32 {
     }
 }
 
+impl ExonumJsonDeserialize for F32 {
+    fn deserialize(value: &Value) -> Result<Self, Box<Error>> {
+        let number = value.as_f64().ok_or("Can't cast json as float")?;
+        Ok(Self::new(number as f32))
```

Comment:
This certainly should do some validation so as not to panic!?

Reply:
Should we? afaik `as` will never panic, can you provide the example of the opposite?

Reply:
`new` panics for NaNs?

Reply:
Nevermind, I see it now.

Reply:
Hm, even if the author of code forgets that new can panic, perhaps we should not have the new method at all? Or make it return an option?

Reply:
:D The whole story about floats is a complete nightmare. But we need them anyway, and new method seems to be well documented. (how am I to read docs?).